backend/domain/VoteHandler.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * A class used to handle any voting related events or calculations for a Room.
+ * This includes tallying votes, and figuring out who the most voted player is.
+ */
+class VoteHandler {
+    constructor() {
+        this.daytimeVoteMap = {};
+        this.mafiaVoteMap = {};
+        this.medicChosenPlayer = null;
+    }
+
+    addDaytimeVote(voter, chosenPlayer) {
+        this.daytimeVoteMap[voter] = chosenPlayer;
+    }
+
+    addMafiaVote(voter, chosenPlayer) {
+        this.mafiaVoteMap[voter] = chosenPlayer;
+    }
+
+    setMedicChosenPlayer(player) {
```

Comment:
Could we remove these setters and getters while we are here? :D

Reply:
These have been removed. Fields from VoteHandler are now accessed directly rather than via setters